modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2699,16 +2701,18 @@ public void testGetValuesList() throws Exception {
 
 		_addObjectEntry(values1);
 
-		valuesList = _objectEntryLocalService.getValuesList(
-			0, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
-			_objectDefinition.getObjectDefinitionId(), null, null, null,
-			QueryUtil.ALL_POS, QueryUtil.ALL_POS, sorts);
+		valuesList = TransformUtil.transform(
+			_objectEntryLocalService.getPrimaryKeyList(
+				0, TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				_objectDefinition.getObjectDefinitionId(), null, null,
+				QueryUtil.ALL_POS, QueryUtil.ALL_POS, sorts),
+			primaryKey -> _objectEntryLocalService.getValues(primaryKey));
 
 		Assert.assertEquals(valuesList.toString(), 1, valuesList.size());
 
 		_assertCount(1);
 
-		_assertObjectEntryValues(29, values1, valuesList.get(0));
+		_assertObjectEntryValues(23, values1, valuesList.get(0));
```

Comment:
Would you mind explaining why we are only asserting 23 instead of 29? Sorry, I didn't get it.

Reply:
Of course. The reason is that now we are using `objectEntryLocalService.getValues()`, which omits certain select expressions. This behavior was also observed in other existing tests that use `_getValuesFromDatabase()` because it internally relies on `objectEntryLocalService.getValues()`. To ensure consistency, we unified this behavior across all related cases.

Reply:
Anyway, I have explained it in the commit message! ;)

Reply:
Thank you!!